src/main/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAge.java

```diff
@@ -54,6 +56,12 @@ protected boolean verifyKey(String key) {
     }
 
     protected int getMaxAge(HttpServletRequest request) throws Exception {
+        if (appConfig().babbage().isLegacyCacheAPIEnabled()) {
+            LegacyProxyMaxAge legacyProxyMaxAge = new LegacyProxyMaxAge();
```

Comment:
Another thing is that, if you don't want to create an object every time, you can declare this as a private static final property of this class, that way it only gets created once, and it's a bit more efficient.

The same goes for `appConfig().babbage().isLegacyCacheAPIEnabled()`, as that never changes either, you may as well save the result in a private static final property.

Reply:
On second thought, making the `LegacyProxyMaxAge` a static property may not be the right thing (`appConfig().babbage().isLegacyCacheAPIEnabled()` would be fine, though). The reason for this is that I cannot remember what would happen if there are multiple concurrent requests to `getMaxAgeValueFromLegacyProxy` (a method that, due to making a network call, is very slow): will they be done in parallel or, given that `LegacyProxyMaxAge` is a static property, will they have to wait until the previous one is finished? You can probably manually test this, but if it's too difficult, you can always just leave it as it is, I don't think it's a big deal.

Reply:
Declaring `appConfig().babbage().isLegacyCacheAPIEnabled()` in a private static final property doesn't work well with the tests because it is initialized as part of the class's instantiation process and I couldn't figure out a way to mock the variable this way